app/app.js

```diff
@@ -67,6 +75,12 @@ const App = Application.extend(Evented, {
         return HS.beacon.identify(userParams);
       });
     }
+  },
+
+  setIntercomUser(user) {
+    this.get('intercom').set('user.name', user.name);
+    this.get('intercom').set('user.email', user.email);
+    // FIXME createdAt?
```

Comment:
we can use `user.first_logged_in_at` i think, and assign it to `createdAt`

Reply:
👍🏻 thanks, I’ve added that now. I’m having a hard time getting anything to build locally so I’ve just pushed and will see how it works on the PR deployment.